src/components/QueryEditor/EditorStack.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import { Stack } from '@grafana/ui';
+import { ThemeSpacingTokens } from '@grafana/data';
+
+export type Direction = 'row' | 'row-reverse' | 'column' | 'column-reverse';
+export type AlignItems =
```

Comment:
Had to add these types here because they're not exported from main grafana. I should probably make a PR to export them for this purpose. 
(CSSProperties['alignItems'] like in the old `Stack` just doesn't work with grafana-ui/stack

Reply:
Do we really need this component? Seems like just a wrapper around the `Stack`. Can we just use `Stack` from `@grafana/ui` where needed?

Reply:
It does add some default behaviour for Query Editors, like 'wrap' and 'row', which overrides the default styles of `Stack` from grafana-ui. This way we don't have to add these every time the Stack is used in the QueryEditor

Reply:
Could we use the same approach as in old Stack component for align items and direction? Like this:
```
direction?: CSSProperties['flexDirection'];
alignItems?: CSSProperties['alignItems'];
```

Reply:
I tried that, but it seems like the new custom Grafana types used in the `Stack` component don't match these and produce a type error. In the end I had to copy them over from Grafana (they're not exported yet so I couldn't use them directly). I've been meaning to make a PR that exports them in main Grafana, will set a reminder